book/templates.rst

```diff
@@ -334,6 +334,11 @@ Inside of these attributes, you can utilize the `jexl`_ syntax for expressing yo
         </properties>
     </template>
 
+.. note::
+
+    Because the ``&`` character cannot be used inside of XML files, you have to use ``AND`` instead of ``&&`` 
```

Comment:
> Because the ``&`` character cannot be used inside of XML files 

this is not technical correct would write as the `&` character need to be escaped in XML files, you can use ``AND`` instead of ``&&``

Reply:
```suggestion
    Because the ``&`` character needs to be escaped inside of XML files, you have to use ``AND`` instead of ``&&`` 
```